SYCL/BFloat16/bfloat16_builtins.cpp

```diff
@@ -1,11 +1,11 @@
-// REQUIRES: cuda
+// RUN: %if cuda %{%clangxx -fsycl -fsycl-targets=%sycl_triple -Xsycl-target-backend --cuda-gpu-arch=sm_80 %s -o %t.out %}
+// RUN: %if cuda %{%GPU_RUN_PLACEHOLDER %t.out %}
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %if cuda %{-Xsycl-target-backend --cuda-gpu-arch=sm_80%} %s -o %t.out
```
Something like this should do the trick. Right?